src/Statement.php

```diff
@@ -154,19 +153,11 @@ public function bindParam($param, &$variable, int $type = ParameterType::STRING,
     /**
      * {@inheritDoc}
      *
-     * @deprecated Statement::execute() is deprecated, use Statement::executeQuery() or executeStatement() instead
-     *
      * @throws Exception
      */
-    public function execute(?array $params = null): Result
+    private function execute(array $params = []): Result
```

Comment:
```suggestion
    private function execute(array $params): Result
```

This private method is never called without parameters. Let's keep things simple. 🙂

Reply:
Oh, you've changed that already. 🚀